tutorials/Basics/Methods-and-Parameters-for-Feature-Detection_Part_2.ipynb

```diff
@@ -0,0 +1,1522 @@
+      "text/plain": [
+       "<Figure size 576x576 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax1 = plt.subplots(ncols=1, figsize=(8, 8))\n",
+    "ax1.imshow(data.data[50], cmap=\"Greys\")\n",
+    "\n",
+    "im1 = ax1.scatter(\n",
+    "    features.where(mask)[\"hdim_2\"], features.where(mask)[\"hdim_1\"], color=\"red\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "163a7045",
+   "metadata": {},
+   "source": [
+    "If the features have the same threshold, tobac keeps the feature with the larger area. Otherwise the feature with the higher treshold is kept."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0bf4f5cb",
+   "metadata": {},
+   "source": [
+    "## Band-Pass Filter for Input Fields via Parameter `wavelength_filtering`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1bf56e78",
+   "metadata": {},
+   "source": [
+    "This parameter can be understood best, when looking at real instead of snythethic data. An example of usage is given [here](../Example_vorticity_tracking_model/Example_vorticity_tracking_model.ipynb)"
+   ]
```

Comment:
Maybe add this small section in `Methods-and-Parameters-for-Feature-Detection_Part_1` instead? It does probably not matter, but I felt it is more logically to mention the ability of wavelength filter together with the gaussian filter, because both input parameters apply some kind of filter on the input data.